Automattic/jetpack
# Fix Boost phpunit tests

Fix broken phpunit tests in Jetpack Boost.

#### Changes proposed in this Pull Request:
* Remove dependency on `WP_Unit`; it requires a higher version of PHP than the Monorepo env. Replaces it with Brain Monkey.

#### Does this pull request change what data or activity we track or use?
no.

#### Testing instructions:
- Run the unit tests with `composer run test-php` from inside the Boost directory. See that it doesn't explode horribly.
